core/src/epicli/data/common/ansible/playbooks/roles/node_exporter/tasks/uninstall-node-exporter-as-daemonset.yml

```diff
@@ -0,0 +1,11 @@
+---
+
+- name: Uninstall Helm release "{{ specification.helm_chart_name }}"
+
+  delegate_to: localhost
+  become: false
+  run_once: true
+
+  environment: { KUBECONFIG: "{{ vault_location }}/../kubeconfig" }
```

Comment:
Recently we reduced the number of tasks where we use `KUBECONFIG` variable. Can't the value be taken from `group_vars/all.yml` and be specified on a playbook level? The same question for the left part of PR.

Reply:
I think in case of just the helm feature we have to keep such construction, but for filebeat and node_exporter we have properly defined non-localhost ansible groups, @atsikham I think you're right. :thinking: Actually in both cases lines
```
environment: { KUBECONFIG: "{{ vault_location }}/../kubeconfig" }
```
can be removed because it's already defined at playbook level. :+1:

Reply:
@rpudlowski93 any thoughts? :hugs:

Reply:
I've just added the kubeconfig env to upgrade playbook level and removed the env from role. Tested, works fine. Small changes also added in order to optimize code